the_hat_game/game.py

```diff
@@ -54,9 +79,11 @@ def remove_same_rooted_words(self, word, word_list):
         cleared_word_list = [w for w in word_list if self.stemmer.stem(w) != root]
         return cleared_word_list
 
-    @staticmethod
-    def remove_non_existing_words(words):
-        existing_words = [w for w in words if len(wordnet.synsets(w)) > 0]
+    def remove_non_existing_words(self, words):
+        if self.used_words is not None:
+            existing_words = [w for w in words if w in self.used_words]
```

Comment:
Maybe this will fix [this strange behaviour](https://github.com/production-ml/the-hat-game/issues/1#issuecomment-968009184), but I would check if that's not `wordnet.synsets` filtering out those words, and can we fix it with using `nltk.corpus.words`.

Reply:
> Let's add words to self.used_words if the later is not empty (in __init__ method).
Oh, sure. Now it seems like a bug. Thank you!

Reply:
> Let's add words to self.used_words if the later is not empty (in __init__ method).

added in https://github.com/production-ml/the-hat-game/pull/24/commits/26e6f0dea108fb1bc3278772596bf488e0e6fb1d

Reply:
> Maybe this will fix this strange behaviour

yes, [adding](https://github.com/production-ml/the-hat-game/commit/26e6f0dea108fb1bc3278772596bf488e0e6fb1d) words for guessing to whitelist fixed [this strange behaviour](https://github.com/production-ml/the-hat-game/issues/1#issuecomment-968009184)

> I would check if that's not wordnet.synsets filtering out those words, and can we fix it with using nltk.corpus.words

`wordnet.synsets` really filter rare English words, but `nltk.corpus.words` do the same (see https://github.com/production-ml/the-hat-game/issues/1#issuecomment-975925670)